lib/linguist/heuristics.yml

```diff
@@ -709,6 +709,10 @@ disambiguations:
   - language: Markdown
     # Markdown syntax for scdoc
     pattern: '^#+\s+(NAME|SYNOPSIS|DESCRIPTION)'
+- extensions: ['.script']
+  rules:
+  - language: URScript
+    pattern: '^\s*end\b'
```

Comment:
This is way too vague and will result in a lot of misclassification. This heuristic needs to be 100% unlikely to match anything other than this language due to the nature of the generic script.

We also need a test for the heuristic.

Reply:
That makes a lot of sense! Thank you for the feedback🙏

I updated it for being more specific and added a test.